I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/story/endings/battle.rpy", line 349, in script
    scene esther death with fade
  File "renpy/common/000window.rpy", line 129, in _window_auto_callback
    _window_hide(auto=True)
  File "renpy/common/000window.rpy", line 111, in _window_hide
    renpy.with_statement(trans)
TypeError: got an unexpected keyword argument 'layer'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/story/endings/battle.rpy", line 349, in script
    scene esther death with fade
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 1253, in execute
    statement_name("scene")
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\ast.py", line 58, in statement_name
    i(name)
  File "renpy/common/000window.rpy", line 129, in _window_auto_callback
    _window_hide(auto=True)
  File "renpy/common/000window.rpy", line 111, in _window_hide
    renpy.with_statement(trans)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\exports\statementexports.py", line 257, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 1581, in do_with
    return self.interact(trans_pause=True,
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2218, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2748, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\displayable.py", line 434, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 480, in visit_all
    callback(self)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\core.py", line 2748, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 491, in per_interact
    self.update()
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\display\screen.py", line 700, in update
    self.screen.function(**self.scope)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\parameter.py", line 455, in apply_arguments
    return parameters.apply(args or (), kwargs or {}, ignore_errors)
  File "C:\Users\seven\Downloads\renpy-8.3.6-sdk\renpy\parameter.py", line 384, in apply
    raise TypeError(
TypeError: got an unexpected keyword argument 'layer'

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.6.25022803
Aperture Dating Simulator 0.2.5alpha
Wed Jun 18 12:13:04 2025
